api/tests/Behat/run-tests-v2.sh

```diff
@@ -10,6 +10,7 @@ confd -onetime -backend env
 
 su-exec www-data php app/console doctrine:fixtures:load --no-interaction
 
+./vendor/bin/behat --config=./tests/Behat/behat.yml --stop-on-failure --profile v2-tests-goutte --tags @v2_standalone
 ./vendor/bin/behat --config=./tests/Behat/behat.yml --stop-on-failure --profile v2-tests-goutte --tags @v2 --list-features | ./vendor/liuggio/fastest/fastest "./vendor/bin/behat --profile v2-tests-goutte --config=./tests/Behat/behat.yml {}"
```

Comment:
Hmmmm - if we end up using the shell script in the makefile is there a way we can still enable passing in the suite tag like we're going with the existing `behat-tests-v2-goutte` command? Just saves having to mess around with different make commands.

Reply:
Er pass.. not sure how we can as this one runs in parallel. When we're wanting to run on individual tags we don't really want liuggio bundling it all up anyway. This for now keep em separate and if we work out a way then we can change it but can't think how now... actually we can do it with bash args but I'll change that in the general fix up and split out parallel builds ticket.